admiral/pkg/clusters/dynamoDB.go

```diff
@@ -71,6 +74,17 @@ type WorkloadData struct {
 	FailedClusters      []string         `json:"failedClusters"`
 }
 
+type DynamicConfigData struct {
+	/*
+		DynamoDB primary key only support string, binary, int.
+		Conversation from binary to bool is not straight forward hence choosing string
+	*/
+	EnableDynamicConfig    string   `json:"enableDynamicConfig"`
```

Comment:
in the example you showed the column names as upper case does it matter during unmarshalling if the column names match the case ?

Reply:
Based upon test I done, it doesn't matter.